### ISSUE TITLE
Helm charts for K8s deployment of Infinity

### ISSUE THREAD

**Issue by indranilr** [michaelfeil/infinity#314]
Although dockerized deployments are supported, an official helm chart would be very handy for K8s deployments. Is this already available ?

**Comment by michaelfeil** [comment]
I agree, but have not used helm recently. There is no helm chart available, potentially the infra folder could be a suitable place?

**Comment by samos123** [comment]
I am working on finishing the KubeAI integration that allows you to deploy Infinity easily on K8s: https://github.com/substratusai/kubeai

PR that's soon to be merged: https://github.com/substratusai/kubeai/pull/197
